src/main.ts

```diff
@@ -22,20 +22,36 @@ async function readReadme(rootDirectory: string, packageName: string) {
   return file
 }
 
-function render(body: string) {
+function highlightRender(body: string) {
```

Comment:
[IMO]
render function is responsible for rendering the filebody, so I think highlightRender is inappropriate.
Render should take the colorized flag as an argument, since render should control whether to decorate.

Reply:
I'll fix it.

Reply:
thank you